xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.h

```diff
@@ -38,12 +39,67 @@ class CDVDSubtitlesLibass
 
   bool DecodeHeader(char* data, int size);
   bool DecodeDemuxPkt(const char* data, int size, double start, double duration);
+
+  /*!
+  * \brief Create a new ASS track with a default ASS style (used on ASS events)
+  * \param useDebugStyle Use style text/color/position for debug info purpose
+  * \return True if has success
+  */
+  bool CreateTrack(bool useDebugStyle = false);
+  /*!
+  * \brief Create a new ASS track based on a SSA buffer
```

Comment:
grammar: `an SSA` instead of `a SSA`, because the leading sound ("*es*") governs whether you use "a" or "an", not the actual letter of the first word.